### PR TITLE
Create templated response article

### PR DESCRIPTION
**Pull request by alexyorke**

### CODE REVIEW

**Review by JohnnyGitH — APPROVED** [review]

File: _data/navigation.yml [hunk 1]
```diff
@@ -26,6 +26,10 @@ docs:
         url: /docs/body-rules
       - title: 'Policies'
         url: /docs/policies
+  - title: 'Orbital response rules'
```
Comment (mariareyesfoci) [inline _data/navigation.yml]: Can we rename this to `Orbital response types`

File: _pages/docs/request-match-rules/templated-responses.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+layout: single
+title: Templated Responses
+permalink: /docs/templated-responses
+sidebar:
+  nav: 'docs'
+classes: wide
+---
+
+Templated responses allow responses to be dynamically created by using Scriban's liquid template API. The responses can be derived from the response's body, and/or a random value using the [faker](https://github.com/bchavez/Bogus) API.
```
Comment (mariareyesfoci) [inline _pages/docs/request-match-rules/templated-responses.md]: can we redo this sentence and not mention the technologies we are using. We want to explain just the feature, not so much what technologies we are using to do this feature.

File: _pages/docs/request-match-rules/templated-responses.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+3. Set the type of the response to "Templated".
+4. As an example, set the body of the response to "\{\{rant.reviews\}\}".
+5. Save the scenario, and upload it to the server.
+6. Make request to the endpoint that contains the templated response.
+
+![Templated Response](../../../assets/images/request-match-rules/add-templated-response.png)
+
+Figure 1. Adding a new templated rule. Note the checkmark next to "Templated".
+
+The server will respond with a rant review string generated from the faker API. When composing a templated scenario body, the body can contain multiple templated responses, such as "\{\{rant.reviews\}\} \{\{person.first\_name\}\}", and can contain text between and around the templates: "\{\{rant.reviews\}\} abc \{\{person.first\_name\}\}". These templates can also be nested inside of JSON. For example, \{"key":"\{\{person.first\_name\}\}"\}.
```
Comment (mariareyesfoci) [inline _pages/docs/request-match-rules/templated-responses.md]: this is a general comment. can we remove faker/bogus and scriban references in the article

File: _pages/docs/request-match-rules/templated-responses.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+All other Scriban operators, for example `string.truncate`, are supported. For example, "\{\{rant.reviews \| string.truncate 6\}\}" returns "The...". To learn more about Scriban, see [https://github.com/lunet-io/scriban](https://github.com/lunet-io/scriban).
+
+### How do templated responses behave?
+
+In order to use templated responses, the request's body must be valid JSON, the Scriban syntax must be valid, the response must be marked as a templated response, and all referenced keys must exist in order to use templated responses. If at least one of these requirements are violated, a `400 Bad Request` will be returned.
+
+When referencing a JSON key in the request body which is...
```
Comment (mariareyesfoci) [inline _pages/docs/request-match-rules/templated-responses.md:37]: the section explaining how you can use the request body in a templated response should have it's own subtitle.

Comment (alexyorke) [reply _pages/docs/request-match-rules/templated-responses.md:37]: What is the sentence just before where the subtitle should be inserted?

File: _data/navigation.yml (hunk 1 above)
Comment (alexyorke) [reply _data/navigation.yml]: Fixed

File: _pages/docs/request-match-rules/templated-responses.md (hunk 2 above)
Comment (alexyorke) [reply _pages/docs/request-match-rules/templated-responses.md]: Fixed

File: _pages/docs/request-match-rules/templated-responses.md (hunk 3 above)
Comment (alexyorke) [reply _pages/docs/request-match-rules/templated-responses.md]: Fixed

File: _pages/docs/request-match-rules/templated-responses.md (hunk 4 above)
Comment (mariareyesfoci) [reply _pages/docs/request-match-rules/templated-responses.md:37]: I was thinking of line 31 and 33. but I see the previous paragraph already talking about request body. so maybe replace the subtitle in line 29

Comment (alexyorke) [reply _pages/docs/request-match-rules/templated-responses.md:37]: Added clarifying subtitle

Comment (alexyorke) [reply _pages/docs/request-match-rules/templated-responses.md:37]: The section has been re-written; if it requires more changes feel free to reopen this

**Review by mariareyesfoci — APPROVED** [review]

**Pull request merged**